### PR TITLE
Package smart_proxy_dns_powerdns

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ekohl** [comment]
I should note that https://packages.debian.org/jessie/ruby-mysql2 is only available on Jessie now.

File: plugins/smart_proxy_dns_powerdns/watch [hunk 1]
```diff
@@ -0,0 +1,2 @@
+version=3
```
Comment (mmoll) [inline plugins/smart_proxy_dns_powerdns/watch]: this file can get deleted

File: plugins/smart_proxy_dns_powerdns/changelog [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ruby-smart-proxy-dns-powerdns (0.1.0-0) stable; urgency=low
```
Comment (mmoll) [inline plugins/smart_proxy_dns_powerdns/changelog]: the revision should be -1

**Comment by mmoll** [comment]
:+1: after changing the revision

**Comment by ekohl** [comment]
Updated and also copied ruby-mysql2 from wheezy backports.

**Comment by mmoll** [comment]
The copied ruby-mysql2 could have some files deleted as we don't run the tests after building the gem normally, but as it's an 1:1 copy I'm fine with it. :+1:

File: plugins/smart_proxy_dns_powerdns/control [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+Homepage: https://github.com/theforeman/smart_proxy_dns_powerdns
+XS-Ruby-Versions: all
+
+Package: ruby-smart-proxy-dns-powerdns
+Architecture: all
+XB-Ruby-Versions: ${ruby:Versions}
+Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, foreman-proxy (>= 1.1.10~rc1), ruby-mysql2
+Recommends: pdns-server
+Description: PowerDNS DNS provider plugin for Foreman's smart proxy
+ PowerDNS DNS provider plugin for Foreman's smart proxy
```
Comment (domcleal) [inline plugins/smart_proxy_dns_powerdns/control]: Duplicate line?

File: plugins/smart_proxy_dns_powerdns/copyright [hunk 4]
```diff
@@ -0,0 +1,28 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: smart_proxy_dns_powerdns
+Source: https://github.com/theforeman/smart_proxy_dns_powerdns
+
+Files: *
+Copyright: 2015 Red Hat Inc. <foreman-dev@googlegroups.com>
```
Comment (domcleal) [inline plugins/smart_proxy_dns_powerdns/copyright]: Perhaps just yourself, or Foreman developers?  Ditto below.

File: plugins/smart_proxy_dns_powerdns/control (hunk 3 above)
Comment (ekohl) [inline plugins/smart_proxy_dns_powerdns/control]: This was generated by gem2deb and the salt plugin has the same thing, so I left it. @mmoll?

File: plugins/smart_proxy_dns_powerdns/copyright (hunk 4 above)
Comment (ekohl) [inline plugins/smart_proxy_dns_powerdns/copyright]: Tbh I didn't really think of a licence and don't know what the policy is. I'm fine with just my name there but I don't care that much as long as the plugin is used and open :)

File: plugins/smart_proxy_dns_powerdns/control (hunk 3 above)
Comment (mmoll) [inline plugins/smart_proxy_dns_powerdns/control]: That's OK, the further line(s) with spaces are the extended description and might actually even be required.

**Comment by ekohl** [comment]
Verified it works on Debian Wheezy and Jessie (using debstaging.theforeman.org). Updated the copyright by just putting my name there.

**Comment by ekohl** [comment]
Failures appear to be because it tries to build ruby-mysql2 for jessie and trusty but they don't need it.

**Comment by mmoll** [comment]
merged, bedankt @ekohl!
building now

**Pull request merged**